stable/mission-control/CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # JFrog Mission-Control Chart Changelog
 All changes to this chart will be documented in this file.
 
+## [5.1.0] - Oct 5, 2020
+* **Breaking**
+* Changed `insightServer.internalHttpPort` to `insightServer.internalPort`
+* Add support for livenessProbe and readinessProbe for all microservices
+* Updated UPGRADE_NOTES.md
```

Comment:
@rahulsadanandan  Can we add what we changed in UPGRADE_NOTES

Reply:
Corrected.